### ISSUE TITLE
Submissions blocked (using our own worker)

### ISSUE THREAD

**Issue by magrichard** [codalab/codabench#605]
Dear @ckcollab,

The workers you helped us to set up in Heidleberg are now working. Thanks!
I was able to successfully run some submissions (competition 218), with workers properly dimensioned to our benchmark.

However, I am still facing the eratic behaviour of some submissions being blocked for hours, without knowing why.

For instance:

Submission 1811 was created 15 hours ago. Looking at the server status, I see that none of the children subtasks has been created.

![Capture d’écran 2020-12-16 à 08 35 19](https://user-images.githubusercontent.com/18577578/102318756-22f56680-3f7a-11eb-8b9d-5bda3a5f4348.png)

When I click on the submission in the regular competition interface or from the competition admin menu, I get a blank field.

![Capture d’écran 2020-12-16 à 08 34 58](https://user-images.githubusercontent.com/18577578/102318849-44565280-3f7a-11eb-8f5a-85729628cff3.png)

**Comment by magrichard** [comment]
After several test, I confirm that the runs are not consistents at all.

See screen capture and debug explanations below as an example.

Submissions  1824 to 1829 were 6 single tasks run againts a given dataset DT5. All were successfully run within few minutes.

Submission 1830 is the submission of DT5 against the 6 tasks at the same time. Children submission were never created and the submission 1830 status is 'running' for 2 hours now. When I click on the submission in the web interface, I get a blank field.

To test wether the problem was related to the number of subtasks, I did some incremential tests:
- submission 1831: DT5 against 3 tasks: 3 children submissions created (1832 to 1834), all successfull within few minutes
- submission 1835: DT5 against 4 tasks: 4 children submissions created (1836 to 1839), all successfull within few minutes
- submission 1840: DT5 against 5 tasks: 5 children submissions created (1841 to 1845), all successfull within few minutes
- submission 1846: DT5 against 6 tasks (same as submission 1830): 6 children submissions created (1846 to 1852), blocked in preparing or submitting status for more than 1h).

Then I tested to run submissions against one or two subtasks that were previously successfull. Everything is now blocked in 'running' or 'preparing' status.

Finally, I tried to cancelled submissions 1846 and 1830 to see if this would unblocked the other submissions but it was not the case.

![Capture d’écran 2020-12-16 à 13 21 50](https://user-images.githubusercontent.com/18577578/102348148-bb531180-3fa1-11eb-9f15-5dde2a946cab.png)
.
![Capture d’écran 2020-12-16 à 13 29 15](https://user-images.githubusercontent.com/18577578/102355090-aed3b680-3fab-11eb-8512-fce57e1e3401.png)

**Comment by ckcollab** [comment]
We're looking into it, I think I have replicated this locally. thanks for the thorough report!

**Comment by ckcollab** [comment]
Believe this is fixed?

![image](https://user-images.githubusercontent.com/2185159/103317768-02e58e80-49e1-11eb-8440-0ab209f834fb.png)

**Comment by magrichard** [comment]
Hi,

I experience some strange behaviour over the week-end.
I believe it is related to this issue, that is why I am reopening it.

I submitted job 2249 on Friday in the benchmark #218. The dataset uploaded for the submission is quite heavy (900Mo) so it took a while to upload. This morning, the status of the job is unclear:

On the regular user interface, it the log black window, it looks like the children job (2250 and 2251) were successfully run. 
![Capture d’écran 2021-01-11 à 09 04 32](https://user-images.githubusercontent.com/18577578/104158234-3666e680-53ed-11eb-9a65-d12cbb6ec888.png)

However in the submission table, it indicated 'running', 
![Capture d’écran 2021-01-11 à 09 03 39](https://user-images.githubusercontent.com/18577578/104158288-4d0d3d80-53ed-11eb-94cb-6f847fefabaa.png)

and when I click on the job, I can bot get any logs, neither download the outputs.


![Capture d’écran 2021-01-11 à 09 04 03](https://user-images.githubusercontent.com/18577578/104158259-42eb3f00-53ed-11eb-9c2e-ce716b085813.png)



I cannot retreive any information on the server_status webpage either.

![Capture d’écran 2021-01-11 à 09 05 40](https://user-images.githubusercontent.com/18577578/104158199-27803400-53ed-11eb-9791-fee63ec7db0d.png)

**Comment by magrichard** [comment]
@ckcollab, if you want me to open a new issue on this, let me know!

**Comment by Didayolo** [comment]
I close this issue as it is old, and workers seems to be working fine now. Feel free to re-open it if needed.
